edward/inferences/variational_inference.py

```diff
@@ -123,6 +123,10 @@ def update(self, feed_dict=None):
 
     sess = get_session()
     _, t, loss = sess.run([self.train, self.increment_t, self.loss], feed_dict)
+    if self.logging and self.n_print != 0:
+      if t == 1 or t % self.n_print == 0:
+          summary = sess.run(self.summarize, feed_dict)
```

Comment:
Sorry it's taken so long to get back to you on this.

I was thinking one might not want to log the summary on every iteration if `n_print != 1`. Happy to move this into line 125, though, if you think it's best.